libcxxabi/CMakeLists.txt

```diff
@@ -269,6 +269,13 @@ add_definitions(-D_LIBCXXABI_BUILDING_LIBRARY)
 # it is being built as part of libcxx.
 add_definitions(-D_LIBCPP_BUILDING_LIBRARY)
 
+# To ensure that all ASan functionalities are built into the binary.
```

Comment:
This also shouldn't be necessary anymore with my suggested `__config_site` changes.